Set/Game/GameViewModel.swift

```diff
@@ -24,17 +38,60 @@ class GameViewModel: ObservableObject {
         isMoreCardAvailable ? .gray : .black
     }
     
+    var player1ButtonColor: Color {
+        if player1.id == whoseTurn?.id {
+            return .green
+        } else if player2.id == whoseTurn?.id {
+            return .gray
+        } else {
+            return .black
+        }
+    }
+    
+    var player2ButtonColor: Color {
+        if player2.id == whoseTurn?.id {
+            return .green
+        } else if player1.id == whoseTurn?.id {
+            return .gray
+        } else {
+            return .black
+        }
+    }
+    
+    var player1: Player {
+        gameModel.firstPlayer
+    }
+    
+    var player2: Player {
+        gameModel.secondPlayer
+    }
+    
+    var whoseTurn: Player?
```

Comment:
After all the Changs this property can became not used.

Reply:
no, I still use it